src/Discussion/Discussion.php

```diff
@@ -461,6 +463,8 @@ public static function setStateUser(User $user)
     protected function setTitleAttribute($title)
     {
         $this->attributes['title'] = $title;
-        $this->slug = Str::slug($title);
+        $app = Application::getInstance();
+        $locale = $app->make(LocaleManager::class)->getLocale() ?? 'en';
```

Comment:
This locale depends on the currently active user. Shouldn't we *always* use either `en` or at least the board's default language here, so that we get consistent results?

Reply:
Good point, I think the default locale would make the most sense.